### PR TITLE
Add install doc

### PR DESCRIPTION
**Pull request by lyd911**

### CODE REVIEW

**Review by leonwanghui — APPROVED** [review]
LGTM

File: docs/en/source/quickstart/install.md [hunk 1]
```diff
@@ -1,3 +1,86 @@
-# Installation
+# Install TinyMS
 
-TODO
+## User install TinyMS
+
+### Pypi
+
+For users owning a clean environment, recommend using [pypi](https://pypi.org/) to install TinyMS after satisfy the following prerequisites. For setting up python environment, [Anaconda](https://www.anaconda.com/products/individual#Downloads) is a good choice.
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:7]: For users who own a clean environment, it is recommend to use pypi to install TinyMS given that the following requirements are meet

Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:7]: For those who don't, Anaconda is a good choice for setting up the python environment

File: docs/en/source/quickstart/install.md [hunk 2]
```diff
@@ -1,3 +1,86 @@
...
+
+Prerequisites  
+
+- Ubuntu: `18.04`
+- Python: `3.7.5`
+
+Chinese Domestic users recommend running the following code setting the mirror site of pypi to solve the downloading issue
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:14]: For China based users it is recommended to run ..... to help with faster download

File: docs/en/source/quickstart/install.md [hunk 3]
```diff
@@ -1,3 +1,86 @@
...
+&& echo "index-url=http://mirrors.aliyun.com/pypi/simple/" >> /root/.pip/pip.conf
+```
+
+```shell
+pip install tinyms==0.1.0
+```
+
+### Docker
+
+For users' environment do not want to change the local env which also cannot satisfy the prerequisites, using [docker](https://www.docker.com/) to install is recommended
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:29]: For those who don't want to affect the local develop environment due to difficulty of meeting the prerequisites

File: docs/en/source/quickstart/install.md [hunk 4]
```diff
@@ -1,3 +1,86 @@
...
+
+- docker: `v18.06.1-ce`
+
+If user want to try the tutorials that are written in `.ipynb` files，please pull `tinyms-jupyter` which jupyter component are installed by default
+
+```shell
+docker pull tinyms/tinyms:0.1.0
+```
+
+If want to try jupyter, run the following command line
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:43]: lack of subject, if who ??

File: docs/en/source/quickstart/install.md [hunk 5]
```diff
@@ -1,3 +1,86 @@
...
+
+Open a browser on the local machine, type in
+
+```URL
+<Your_external_IP_address>:8888
+```
+
+Example: `159.138.7.105:8888`, the default password is `tinyms`，then user can log in to `jupyter`
+
+## Developer install TinyMS
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md]: TinyMS Installation For Experienced Developers

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
Section titles needs to be modified, a lot of the sentences either lack of subject or not properly composed

File: docs/en/source/quickstart/install.md [hunk 6]
```diff
@@ -1,3 +1,88 @@
-# Installation
+# Install TinyMS
 
-TODO
+## User install
+
+### Pypi
+
+For users who own a clean environment, it is recommend to use [pypi](https://pypi.org/) to install TinyMS given that the following requirements are meet. For those who don't, [Anaconda](https://www.anaconda.com/products/individual#Downloads) is a good choice for setting up the python environment.
```
Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:7]: `recommend` or `recommended`?

File: docs/en/source/quickstart/install.md [hunk 7]
```diff
@@ -1,3 +1,88 @@
...
+pip install tinyms==0.1.0
+```
+
+### Docker
+
+For those who don't want to affect the local develop environment due to difficulty of meeting the prerequisites, using [docker](https://www.docker.com/) to install is recommended
+
+- docker: `v18.06.1-ce`
+
+If user want to try the tutorials that are written in `.ipynb` files，please pull jupyter version of TinyMS which jupyter component are installed by default
```
Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:33]: s/want/wants

Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:33]: s/which/where

Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:33]: s/component/components

File: docs/en/source/quickstart/install.md [hunk 8]
```diff
@@ -1,3 +1,88 @@
...
+
+```URL
+<Your_external_IP_address>:8888
+```
+
+Example: `188.8.8.88:8888`, the default password is `tinyms`，then user can log in to `jupyter`
+
+## TinyMS Installation For Experienced Developers
+
+For developers who want to develope based on TinyMS, install from source
```
Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:60]: s/develope/develop

File: docs/en/source/quickstart/install.md [hunk 9]
```diff
@@ -1,3 +1,88 @@
...
+
+```shell
+sudo apt-get install -y libssl-dev
+git clone https:github.com/tinyms-ai/tinyms.git
```
Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:64]: The link should be `https://github.com/tinyms-ai/tinyms.gi`

File: docs/zh_CN/source/quickstart/install.md [hunk 10]
```diff
@@ -1 +1,89 @@
...
+
+例如 `188.8.8.88:8888`，之后在弹出的页面中，密码输入`tinyms`，就可以远程登录`jupyter`了
+
+## 源码安装
+
+想针对TinyMS进行开发的开发者，可以通过源码安装
+
+```shell
+sudo apt-get install -y libssl-dev
+git clone https:github.com/tinyms-ai/tinyms.git
```
Comment (leonwanghui) [inline docs/zh_CN/source/quickstart/install.md:65]: Same comments

**Review by leonwanghui — CHANGES_REQUESTED** [review]

File: docs/en/source/quickstart/install.md [hunk 11]
```diff
@@ -4,14 +4,14 @@
```
Comment (hannibalhuang) [inline docs/en/source/quickstart/install.md:4]: ## Installation For Beginners

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
nixes for section titles

File: docs/en/source/quickstart/install.md [hunk 12]
```diff
@@ -1,3 +1,88 @@
...
+
+Open a browser on the local machine, type in
+
+```URL
+<Your_external_IP_address>:8888
+```
+
+Example: `188.8.8.88:8888`, the default password is `tinyms`，then user can log in to `jupyter`
+
+## TinyMS Installation For Experienced Developers
```
Comment (leonwanghui) [inline docs/en/source/quickstart/install.md:58]: Please remove `TinyMS`

**Review by leonwanghui — APPROVED** [review]
LGTM

**Review by hannibalhuang — APPROVED** [review]
LGTM

**Pull request merged**